src/main/java/com/google/codeu/data/CohortDatastore.java

```diff
@@ -51,4 +55,14 @@ public Cohort getCohort(Entity cohortEntity) {
         return new Cohort(cohortID, cohortName);
     }
 
+    public List<Cohort> getCohorts() {
+        List<Cohort> cohorts = new ArrayList<>();
+        Query query = new Query("Cohort");
+        PreparedQuery results = cohortDatastore.prepare(query);
+        for(Entity entity : results.asIterable()) {
+            cohorts.add((Cohort) entity.getProperty("cohortName"));
```

Comment:
Remember the query returns an entity so you'll have to turn that into a cohort. The value returned by `entity.getProperty("cohortName")` is a String so you'll get a cast error when attempting to cast it to a Cohort. 

Since you have the getCohort method you can do the following instead:

```suggestion
            cohorts.add(getCohort(entity));
```